participants/1475Cloud.md

```diff
@@ -0,0 +1,120 @@
+# <1475Cloud>
+
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+
+- Respond to add your response in each section below
+
+
+## (0) Project Name and Team Slack Handles
+
+*Please tell us your project's name and the Filecoin Slack handles for your teammates.*
+
+1475Cloud
+
+https://filecoinproject.slack.com/team/UPC9BT2BG
+
+
+## (1) Brief Project Description
+
+*Add your project's description here.*
+
+Providing one-stop service of FIlecoin storage and retrieval for enterprise and individual users
+
+
+## (2) Link to Project UI
+
+*Please include a link to your project application or UI. It does not need to be super polished or completely functional. We will be reviewing this link periodically to verify that your project is eligible for final rewards.*
+
+yun.1475ipfs.com
+
+
+## (3) What does your application/UI do?
+
+*Give us a brief description of what your application/UI does and how users are intended to interact with it.*
+
+Uploading documents to Filecoin network, downloading from Filecoin network and managing online
+
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*You can reference the [list of curated datasets in this repo](https://github.com/filecoin-project/slingshot/blob/master/datasets.md). If you'd like to use a dataset that is not on that list, please nominate your chosen dataset by updating the curated datasets table in this PR. If nominating a dataset, do you own or have exclusive rights to the data you plan to use?*
+
+Not curated dataset 
+
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect? Do you own or have exclusive rights to the data you plan to use? Is the data license available online? Does the data contain any sensitive or illegal information?*
+
+sql catalogue/unstructured content
```

Comment:
Can you provide some more information on the dataset?  Or describe the purpose of the data that enterprise and individual users may complete?

Reply:
Sure, I would like to add some detail about our dataset then add a new PR.

Reply:
Can you link the new PR when it is opened to this PR as well?